packages/synapse-react-client/src/components/SynapseTable/SendToCavaticaConfirmationDialog.tsx

```diff
@@ -44,7 +44,7 @@ export default function SendToCavaticaConfirmationDialog(
     fileNameColumnName,
     fileVersionColumnName,
     cavaticaHelpURL,
-    cavaticaConnectAccountURL = 'https://help.eliteportal.org/help/limited-data-commons#LimitedDataCommons-GainingAccess',
```

Comment:
Should this prop now be required? Link's `href` can now be undefined which I think is not desirable

Reply:
Good point

Reply:
If link is undefined, then we should not show the portal-branded "Click here for instructions" link